### PR TITLE
[ios] Added map status check to PP action buttons state.

### PR DESCRIPTION
**Pull request by igrechuhin**

### CODE REVIEW

**Comment by VladiMihaylenko** [comment]
LGTM

File: iphone/Maps/Classes/MWMPlacePageButtonCell.mm [hunk 1]
```diff
@@ -1,60 +1,86 @@
...
+- (void)refreshButtonEnabledState
+{
+  if (self.countryId == kInvalidCountryId)
+  {
+    self.titleButton.enabled = YES;
+  }
+  else
+  {
+    NodeAttrs nodeAttrs;
+    GetFramework().Storage().GetNodeAttrs(self.countryId, nodeAttrs);
```
Comment (bykoianko) [inline iphone/Maps/Classes/MWMPlacePageButtonCell.mm]: GetNodeStatuses?

Comment (igrechuhin) [inline iphone/Maps/Classes/MWMPlacePageButtonCell.mm]: OK

**Comment by bykoianko** [comment]
LGTM

**Pull request merged**